web/app/templates/search.hbs

```diff
@@ -0,0 +1,167 @@
+{{!--
+
+ Copyright 2016 LinkedIn Corp.
+
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ use this file except in compliance with the License. You may obtain a copy of
+ the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ License for the specific language governing permissions and limitations under
+ the License.
+
+--}}
+
+<div class="nav-extension"></div>
+
+<div class="container" role="main">
+
+    <div class="shadow box search-box">
+        <form id="search-form" {{action 'search' on="submit"}}>
+
+            <table class="search-table">
```

Comment:
These are the form inputs arranged in the form of a grid. I've used table to maintain the proper indentation and spacing.